deploy/helm/environments/prod/taskforce-connector.yaml.gotmpl

```diff
@@ -0,0 +1,21 @@
+replicaCount: 1
+
+# This sets the container image more information can be found here: https://kubernetes.io/docs/concepts/containers/images/
+image:
+  repository: ghcr.io/taskforcesh/taskforce-connector@sha256
+  # This sets the pull policy for images.
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "a5fd30082e405d17646c0084db24333bef6b792f08327a64851ea57681988347"
+
+params:
+    TASKFORCE_CONNECTION: "incentives-prod"
+    TASKFORCE_TEAM: "devops"
+    REDIS_HOST: "redis-master"
```

Comment:
DISABLE_SCHEDULED_CALCULATIONS=true
 DISABLE_SCHEDULED_SNAPSHOT=true

Can you add above env vars which disable scheduled jobs which we don't want to run until env is ready